pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -98,11 +98,12 @@ func NewCmdNewBuild(fullName string, f *clientcmd.Factory, in io.Reader, out io.
 	cmd.Flags().StringSliceVarP(&config.ImageStreams, "image", "i", config.ImageStreams, "Name of an image stream to to use as a builder.")
 	cmd.Flags().StringSliceVar(&config.DockerImages, "docker-image", config.DockerImages, "Name of a Docker image to use as a builder.")
 	cmd.Flags().StringVar(&config.Name, "name", "", "Set name to use for generated build artifacts")
+	cmd.Flags().StringVar(&config.To, "to", "", "Push built images to this image stream tag (or Docker image repository if --to-docker is set)")
```

Comment:
> all other descriptions have dot at the end and this doesn't have it. May be add it?

Yes, well noted.

> BTW, I would name this option --to-stream to make name more obvious (IMHO).

The `--to` option can specify an image stream tag or a docker image reference:

`oc new-build <source> --to-docker --to my/docker:ref`
`oc new-build <source> --to my/image:stream`